lib/vector/Vlib/break_polygons.c

```diff
@@ -130,12 +131,30 @@ void Vect_break_polygons_file(struct Map_info *Map, int type,
 
     filename = G_tempfile();
     fd = open(filename, O_RDWR | O_CREAT | O_EXCL, 0600);
+    if (fd < 0) {
+        G_free(filename);
+        G_fatal_error(_("Failed to create temporary file: %s"),
+                      strerror(errno));
+    }
     RTree = RTreeCreateTree(fd, 0, 2);
-    remove(filename);
+    if (remove(filename) != 0) {
+        G_warning(_("Failed to remove temporary file: %s"), filename);
+    }
```

Comment:
```suggestion
    (void)remove(filename);
```

As the above open may fail, this may too.

Reply:
Done